tools/releaseBuild/azureDevOps/templates/nuget.yml

```diff
@@ -222,9 +222,36 @@ jobs:
       Write-Verbose -Verbose -Message $fileContent
 
       Copy-Item -Path "$packagePath\SHA512SUMS" -Destination '$(System.ArtifactsDirectory)\signed\' -verbose
-      Copy-Item -Path "$packagePath\SHA512SUMS" -Destination '$(System.ArtifactsDirectory)\signed\globaltool\' -verbose
     displayName: Generate checksum file
```

Comment:
```suggestion
    displayName: Generate checksum file for packages
```

Reply:
@adityapatwardhan Can you please address this comment? I can merge afterwards.